Neos.ContentRepository.Core/Classes/Feature/ContentStreamHandling.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\Feature;
+
+use Neos\ContentRepository\Core\CommandHandlingDependencies;
+use Neos\ContentRepository\Core\EventStore\Events;
+use Neos\ContentRepository\Core\EventStore\EventsToPublish;
+use Neos\ContentRepository\Core\Feature\ContentStreamClosing\Event\ContentStreamWasClosed;
+use Neos\ContentRepository\Core\Feature\ContentStreamClosing\Event\ContentStreamWasReopened;
+use Neos\ContentRepository\Core\Feature\ContentStreamCreation\Event\ContentStreamWasCreated;
+use Neos\ContentRepository\Core\Feature\ContentStreamForking\Event\ContentStreamWasForked;
+use Neos\ContentRepository\Core\Feature\ContentStreamRemoval\Event\ContentStreamWasRemoved;
+use Neos\ContentRepository\Core\SharedModel\Exception\ContentStreamAlreadyExists;
+use Neos\ContentRepository\Core\SharedModel\Exception\ContentStreamDoesNotExistYet;
+use Neos\ContentRepository\Core\SharedModel\Exception\ContentStreamIsClosed;
+use Neos\ContentRepository\Core\SharedModel\Exception\ContentStreamIsNotClosed;
+use Neos\ContentRepository\Core\SharedModel\Workspace\ContentStreamId;
+use Neos\ContentRepository\Core\SharedModel\Workspace\ContentStreamStatus;
+use Neos\EventStore\Model\EventStream\ExpectedVersion;
+
+trait ContentStreamHandling
```

Comment:
todo naming ;) and internal annotation ;)

Reply:
possibly even model it as own class?

- `ContentStreamHandler`, we still have the _real_ handler but that will be removed soon
- `ContentStreamEventsToPublish`
- ... idk

Reply:
good enough :)